App/views.py

```diff
@@ -0,0 +1,26 @@
+from django.http import HttpResponse
+from django.shortcuts import render
+import json
+import requests
+
+def main(request):
+    if(request.method=='POST'):
+        usr = request.POST['username']  #Here, we get the username fetched from html page
+        usr = usr.replace("&","%26")
```

Comment:
But now the username will look weird when you print it out :)

Best practice is to sanitize close to the input, escape close to the output. (Sanitize = reject invalid input. Here the API takes care of that so you don't have to. Escape = convert the variable to a safe format. "safe" depends on context (you escape differently for text in a HTML document, for an URL part, for a Javascript variable...) so it has to be done when you already know how you'll use it; ideally in the template. Most templating languages (including Django templates) support that via some special notation.

Reply:
I have tried to make the variable safe, by escaping &. Is It ok?